### ISSUE TITLE
Replies via Email: SparkPost or Mandrill?

### ISSUE THREAD

**Issue by raamdev** [websharks/comment-mail-kb#28]
Published: http://comment-mail.com/kb-article/replies-via-email-sparkpost-or-mandrill/

**Comment by RealDavidoff** [comment]
Raam, improvement possible such as, why is MC Mandrill now "deprecated", what makes you feel Sparkpost will stay free for long (Mandrill was earlier free too, and is still for MC account holders is my understanding), anything other than spam stuff to consider for the decision,...

**Comment by raamdev** [comment]
@RealDavidoff The MailChimp Mandrill integration was deprecated because we'd rather put our energy and attention on providing really good support for one integration, rather than trying to keep up with changes across multiple integrations. We haven't seen much of a demand for Mandrill, and in fact we saw a flood of people switch away from Mandrill once a paying MailChimp account was required (in fact, I think the Mandrill add-on costs an additional monthly fee, on top of whatever you're already paying MailChimp). 

So the Mandrill integration was marked as 'deprecated' when we added SparkPost. The integration will remain in Comment Mail Pro for the foreseeable future and if there's enough demand to keep the integration around, I'll gladly do so. But not trying to support multiple integrations means more time and energy can go into fixing bugs and making the rest of the plugin better. :-)
